### DISCUSSION TITLE
对于重定向的网页要怎么处理

### DISCUSSION THREAD

**Discussion by sunjx97** [qd-today/qd#232] [Q&A]
像下面这样的
`<html><body><script language="javascript" type="dbf5caefed854381d6766ac0-text/javascript">window.location="/index" +  "."+   "p"  +   "hp?actio"   +  "n=a"   +   "ddbon"   +"us&__SAK"+"URA=35a"+   "2a7382"+"7dff55d2"   +"21d447d" + "8" + "44831"  +"ece1" +   "64"   +  "6"   + "75"   + "22" +  "10"  +  "_56697" +  "10"   +   "8";</script><script src="[/cdn-cgi/scripts/7d0fa10a/cloudflare-static/rocket-loader.min.js](https://github.com/qiandao-today/qiandao/discussions/new)" data-cf-settings="dbf5caefed854381d6766ac0-|49" defer=""></script></body></html>`

**Comment by a76yyyy** [comment]
可以通过正则表达式匹配提取出完整的网址

**Reply by jacrobo** [reply]
然后呢？提取后是否有哪个api可以支持跟踪接下来的response？
